app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,9 +1,9 @@
-<%- @page_title = 'What to expect… Civilians' -%>
+<%- @page_title = "#{abbr_tag('dod')} Civilian Employees" -%>
```

Comment:
I think this is a confusing page title on its own. It makes more sense under the context of Moving Guide > Sub-item